### PR TITLE
Bump BL version to 1.5.6

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Bumps BL to `1.5.6`
* Uses Nick's fix of the `isLarge` prop warning
* Doesn't include any of the migration to GCB, that's in a separate branch
* Updates almost all `npm` packages
* No new feature
* Heres [the diff](https://github.com/getblocklab/block-lab/compare/v1.5.5...bump/1.5.6) since the last release. A lot of it is new JS component and e2e tests, which of course aren't deployed to wp.org.

#### Testing instructions
1. `rm js/editor.blocks.asset.php js/editor.blocks.js css/blocks.editor.css` # remove built files, to ensure this is using the most recently built versions, not falling back to previous builds
2. `npm install && npm run dev`
3. Regression testing of plugin, especially in the block editor. It looked good in my testing.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "root": true,
   "extends": [
-    "plugin:@wordpress/eslint-plugin/recommended",
+    "plugin:@wordpress/eslint-plugin/recommended-with-formatting",
```
Comment (kienstra) [inline .eslintrc:4]: Prevents [having to apply](https://github.com/getblocklab/block-lab/pull/673/commits/5186c4242599b98b927e16a6689d382d6aac96c7) all of the Prettier rules, which are mainly formatting

**Comment by kienstra** [comment]
Hi @lukecarbis and @RobStino,
Could you please test this? It'd be good to release this to wp.org by your Tuesday, a day before the planned WP 5.5 release.

Thanks!

**Review by lukecarbis — APPROVED** [review]
This looks good. Everything test out perfectly. Want me to release once this is merged?

**Comment by kienstra** [comment]
Hi @lukecarbis,
Thanks a lot for reviewing and approving this! 

Sure, it'd be great if you could release it.

**Comment by RobStino** [comment]
Smoke test is good, but I did notice something odd for the first time. Not related to this release, so not a blocker, but worth calling out. Details in vid
https://d.pr/v/HCgygB

**Comment by kienstra** [comment]
Thanks for bringing that up, that's a strange one.

**Comment by kienstra** [comment]
Opening a PR for it.

**Pull request merged**